nimbus.fml.yaml

```diff
@@ -350,6 +397,15 @@ types:
           description: A favicon icon
         letter:
           description: A letter icon
+    MessageSurfaceId:
+      description: >
+        For messaging, we would like to have a message tell us which surface its associated with.
+        This is a label that matches across both Android and iOS.
+      variants:
+        NewTabCard:
+          description: This is the card that appears at the top on the Firefox Home Page.
+        Unknown:
+          description: A message has NOT declared its target surface.
```

Comment:
Nit: I'm not sure what this is used for.

Suggest removing it until we know a little more about it.